assets/pages/new-contributions-landing/components/ContributionThankYou.jsx

```diff
@@ -62,14 +61,14 @@ function ContributionThankYou(props: PropTypes) {
         </section>
       ) : null}
       <MarketingConsent />
-      <ButtonWithRightArrow
-        componentClassName={classNameWithModifiers('confirmation', ['backtothegu'])}
-        buttonClassName={classNameWithModifiers('button', ['wob'])}
-        accessibilityHintId="accessibility-hint-return-to-guardian"
-        type="button"
-        url="https://www.theguardian.com"
-        buttonCopy="Return to The Guardian&nbsp;"
-      />
+      <div className="confirmation confirmation--backtothegu">
```

Comment:
I wonder if you could just put the modifier classes directly on the `CtaLink`? then you wouldn't need the direct child selector in the CSS (we tend to want to keep our selectors just flat classes as it makes overriding things easier)

Reply:
Have added `--backtothegu` as class modifier on `component-cta-link` and updated css to set font size on this.